system/doc/reference_manual/processes.xml

```diff
@@ -109,23 +185,508 @@ spawn(Module, Name, Args) -> pid()
+	  <i>directly visible Erlang resources</i></seeguide> used by the
+	  process have been released.
+	</item>
+
+	<tag><c>monitor</c></tag>
+	<item>
+	  Sent to a process or a port when monitoring via the
+	  <seemfa marker="erts:erlang#monitor/2">monitor/2</seemfa> BIF.
+	</item>
+
+	<tag><c>demonitor</c></tag>
+	<item>
+	  Sent to a process or a port when deactivating a monitor via the
+	  <seemfa marker="erts:erlang#demonitor/1">demonitor/1</seemfa> BIF
+	  or when a process monitoring another process terminates.
+	</item>
+
+	<tag><c>down</c></tag>
+	<item>
+	  Sent to a process or a port when a
+	  <seeguide marker="#monitors">monitored process or port
+	  terminates</seeguide>. Upon reception of a <c>down</c> signal the
+	  signal is converted into an Erlang message and moved into the
+	  message queue if the monitor is still active. The signal is sent
+	  after all <seeguide marker="#visible-resources"><i>directly visible
+	  Erlang resources</i></seeguide> used by the process have
+	  been released.
+	</item>
+
+	<tag><c>change</c></tag>
+	<item>
+	  Sent by the <seeguide marker="#runtime-service">clock
+	  service</seeguide>  on the local runtime system, when the
+	  <seemfa marker="erts:erlang#time_offset/0">time offset</seemfa>
+	  changed, to processes which have
+	  <seemfa marker="erts:erlang#monitor/2">monitored the
+	  <c>time_offset</c></seemfa>.
+	</item>
+
+	<tag><c>group_leader</c></tag>
+	<item>
+	  Sent to a process when changing the group leader of another process
+	  via the
+	  <seemfa marker="erts:erlang#group_leader/2">group_leader/2</seemfa>
+	  BIF.
+	</item>
+
+	<tag><c>spawn_request</c></tag>
+	<item>
+	  Sent when <seeguide marker="#process-creation">creating a new
+	  process</seeguide> via one of the spawn BIFs. The signal is sent to
+	  the <seeguide marker="#runtime-service">spawn service</seeguide> on
+	  the runtime system of the node where the new process is to be
+	  spawned.
+	</item>
+
+	<tag><c>spawn_reply</c></tag>
+	<item>
+	  Sent by the <seeguide marker="#runtime-service">spawn
+	  service</seeguide> on the runtime system of a node creating a
+	  process as a response to a <c>spawn_request</c> signal.
+	</item>
+
+	<tag><c>alive_request</c></tag>
+	<item>
+	  Sent to a process when checking if a process is alive via the
+	  <seemfa marker="erts:erlang#is_process_alive/1">is_process_alive/1</seemfa>
+	  BIF.
+	</item>
+
+	<tag><c>process_info_request</c></tag>
+	<item>
+	  Sent to a process when requesting information about another process
+	  via the
+	  <seemfa marker="erts:erlang#process_info/2">process_info/2</seemfa>
+	  BIF.
+	</item>
+
+	<tag><c>gc_request</c></tag>
+	<item>
+	  Sent to a process when requesting a garbage collection of another
+	  process via the
+	  <seemfa marker="erts:erlang#garbage_collect/1">garbage_collect/1,2</seemfa>
+	  BIFs. This signal is also sent to the caller itself when requesting
+	  the garbage collection to be performed asynchronously when using
```

Comment:
```suggestion
	  the garbage collection to be performed asynchronously using
```

Reply:
I think the meaning of this sentence changes to something it is not supposed to say when removing "when". What about:

This signal is also sent to the caller itself when it requests the garbage collection to be performed asynchronously by passing the `{async, RequestId}` option to `garbage_collect/2`.